### ISSUE TITLE
va-input-telephone - Accessibility review: Error state issues

### ISSUE THREAD

**Issue by amyleadem** [department-of-veterans-affairs/vets-design-system-documentation#4222]
## Issue Description
I discovered the following issues with error state when testing the [va-input-telephone](https://design.va.gov/storybook/?path=/docs/components-va-input-telephone--docs) component:

### Error state
1.  **Issue:** The text input for Phone number moves into an error state immediately on first focus, before the user has a chance to enter anything. This error on focus interrupts screen reader readouts. 
    - Recommendation: Run validation later in the interaction, either when focus is moved out of the input or on submit.
    - Related WCAG: 3.3.1 error identification
    - Tested with: keyboard and mouse interaction in Chrome
2. ~**Issue:** (possibly out of scope/non-blocking because it appears to be a pre-existing issue with combo box) Country combo box does not return expected results after typing. For example, If I type in the country code ( "+1") I receive the error: "Please choose a country". Since "+1" is technically the correct content here, this feels like it shouldn't trigger an error state.~
- ~Recommendation: Allow the user to  enter the country code number without requiring them to type the country name.~
- ~Related WCAG: 3.3.1 error identification~
Removing because this item is a duplicate of issue https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/4226, and should be resolved with PR https://github.com/department-of-veterans-affairs/component-library/pull/1630

3. ~**issue:** (potentially non-blocking) The phone input returns a different error message if I enter alpha characters ("aaa") vs numeric ("123"). The increased specificity of the numeric input would be helpful for users who enter non-numeric characters as well.~ 
    - ~Related WCAG: 3.3.1 error identification~
    - ~Recommendation: Wondering if the error messages can be consistent by making the alpha character entry receive the more specific error that the numeric entry receives.~ Removing this item because it has been resolved

**Comment by amyleadem** [comment]
## Notes
We also should remove "please" from our error messaging to match existing guidance

**Comment by amyleadem** [comment]
Update: Item 2 should be fixed by PR https://github.com/department-of-veterans-affairs/component-library/pull/1602
